tasks/extract-binaries-from-image/extract-binaries-from-image.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: extract-binaries-from-image
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that extracts binaries to be released on github.com from an image
+  params:
+    - name: image_binaries_path
+      type: string
+      description: Path inside the image where the binaries to extract are stored
+      default: "releases"
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      default: snapshot.json
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used for storing the binaries
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot is stored. The extracted binaries will be stored here as well.
+  steps:
+    - name: extract-binaries-from-image
+      image: quay.io/redhat-appstudio/release-service-utils:447ea0580a2cdd48b4091e1df86fab5c3f86d01c
+      script: |
```

Comment:
would be nice if you add the script shebang.

Reply:
Added! Also I added the `-ex` to exit if there is an error and for debugging purposes! Let me know if that's OK

Reply:
Note that if the shebang is missing, tekton will use this by default:
```sh
#!/bin/sh
set -e
```
https://tekton.dev/docs/pipelines/tasks/#running-scripts-within-steps

So personally I don't mind omitting it. But having it is always fine too.